turbo.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "baseBranch": "origin/main",
```

Comment:
Nice, this is an issue I was looking into resolving. Glad to see this change occurring